google/auth/jwt.py

```diff
@@ -59,8 +59,18 @@
 from google.auth import exceptions
 import google.auth.credentials
 
+try:
+    from google.auth.crypt import es256
+except ImportError:  # pragma: NO COVER
+    es256 = None
```

Comment:
Just to be clear on the flow here... 

To use `es256`, a user will include `from google.auth.crypt import es256`, at which point an error is thrown and they can import the dependent library. Otherwise, everything functions as expected.

Reply:
yes this is the behavior